### PR TITLE
Update themis sd env vars

### PR DESCRIPTION
**Pull request by morgansaycea**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: apps/met/themis-fe/test.yaml [hunk 1]
```diff
@@ -7,14 +7,48 @@ metadata:
...
+        THEMIS_GATEWAY_JKSLOCATION: /opt/jks/ThemisAzureKeyStore.jks
+        DAC_KEY_EXPIRE_TIME_SECONDS: 1800
+        DAC_NUMBER_OF_INSTANCES: 5
+        CONFISCATION_SCHEMAS: 110,45,97,116,66,109
+        MAINTENANCE_SCHEMAS: 111,67,44
+        THEMIS_DAC_LISTENERSENDERID: Themis_GW
+        THEMIS_DAC_LISTENERQUEUENAME: LIBRAGOB_WRITERQUEUE
+        THEMIS_DAC_WRITERQUEUENAME: LIBRAGOB_READERQUEUE
+        SPRING_JMS_SERVICEBUS_IDLE-timeout: 30000
+        KV_CONFIG_TREE: /mnt/secrets/libragob-test-kv
```
Comment (thomast1906) [inline apps/met/themis-fe/test.yaml:30]: Is this value needed? Example reference to where in yaml - although are you using this varlabe there?
https://github.com/hmcts/draft-store/blob/master/src/main/resources/application.yaml#L15-L16

File: apps/met/themis-fe/themis-fe.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ spec:
       aadpodidbinding: met-schema-binding
     nodejs:
       ingressHost: 'gobgateway.test.platform.hmcts.net'
-      image: 'sdshmctspublic.azurecr.io/themis/themis_sb_azure:1.0.0'
+      image: 'sdshmctspublic.azurecr.io/themis/themis_sb_azure:1.0.1'
```
Comment (thomast1906) [inline apps/met/themis-fe/themis-fe.yaml:15]: Maybe worth us talking about automated image deployments rather than manual PR each time for image update

**Review by thomast1906 — APPROVED** [review]
Review comments - rest looks fine

File: apps/met/themis-fe/themis-fe.yaml (hunk 2 above)
Comment (TomKimber) [reply apps/met/themis-fe/themis-fe.yaml:15]: Discussed with CGI.

File: apps/met/themis-fe/test.yaml (hunk 1 above)
Comment (TomKimber) [reply apps/met/themis-fe/test.yaml:30]: Discussed with CGI but would required a Key Vault redeployment..

**Review by TomKimber — APPROVED** [review]

**Pull request merged**